server/www/classes/communication/serverMessageReading.js

```diff
@@ -67,6 +67,20 @@ function isValidUnitChar(character)
     return (character<'0' || character>'9') && character!=':' && character!='.' && character!=',' && character!=';' && character!='|' ;
 }
 
+function isTextFormatTelemetry(msg, startIdx, endIdx, flags)
+{
+    if (flags.includes("unit"))
+        return false;
+
+    for (let i = startIdx; i < endIdx; i++)
```

Comment:
This loop might be simplified with the .includes() and .some() functions:
Like `["*","g"].some(c=>"sdkgjnmzgh".includes(c)); // = true`